utils/Util.js

```diff
@@ -684,6 +696,19 @@ class Util {
 		if (data instanceof Array) return data.join(', ');
 		return String(data);
 	}
+
+	/**
+	 * Takes the provided language code, checks it against the SUPPORTED_LANGS-table 
+	 * and if not found calls the getLanguage-method to read from DB or return default
+	 * @param {string} lang Language code (for example provided by arguments)
+	 * @param {ContextMessage} ctx The context (for bot and guild.id)
+	 * @returns {string} A valid language code
+	 */
+	static async getValidLanguageCode(lang, ctx){
```

Comment:
I would better see this method in the Sebedius class but the change can wait for a later update.

Reply:
I can move the method there if you'd like. The question is, if i should just change the "getLanguage" method to support an optional language code argument or put them side by side, as they maybe accomplish different things.

Reply:
Let's keep two different methods!

Reply:
Alright, i will make the changes and push them propably later today.